cli/translator.test.js

```diff
@@ -0,0 +1,137 @@
+import fs from "fs/promises"
+import { expect } from "chai"
+import o from "outdent"
+import translator from "./translator.js"
+
+import mkdirp from "../lib/mkdirp.js"
+import tmpdir from "../test/tmpdir.js"
+
+let dir
+beforeEach(async () => {
+  dir = await tmpdir()
+})
+
+afterEach(async () => {
+  await fs.rm(dir, { recursive: true })
+})
+
+describe(`translator`, () => {
+  describe(`no path given`, () => {
+    beforeEach(async () => {
+      await fs.writeFile(`manifest.json`, o`whatever`)
+    })
+
+    it(`return empty string with undefined`, async () => {
+      const path = undefined
+      expect(await translator(path)).to.be.equal(``)
+    })
+
+    it(`return empty string with null`, async () => {
+      const path = null 
+      expect(await translator(path)).to.be.equal(``)
+    })
+  })
+
+  describe(`app path given`, () => {
+    beforeEach(async () => {
+      await fs.writeFile(`manifest.json`, o`whatever`)
+    })
+
+    it(`return namespace after path`, async () => {
+      expect(await translator(`app/east/asia`)).to.be.equal(`east/asia`)
+    })
+
+    it(`return namespace after path`, async () => {
+      expect(await translator(`app/east`)).to.be.equal(`east`)
+    })
+
+    it(`return '' with only app`, async () => {
+      expect(await translator(`app`)).to.be.equal(``)
+    })
+
+  })
+
+  describe(`invalid path and cwd`, () => {
+    it(`should throw error`, async () => {
+      await mkdirp(`lib`)
+      process.chdir(`lib`)
```

Comment:
Translator should never use the current directory. So, we should not need to set it neither.

It is fine to have a requirement that the path given to Translator is an absolute path (that is anyway what process.cwd() will give).

Reply:
DONE

Reply:
The `process.chdir()` is still there. It is not needed

Reply:
DONE